test/i965_avce_context_test.cpp

```diff
@@ -115,7 +119,7 @@ TEST_P(AVCEContextTest, RateControl)
         config = createConfig(profile, entrypoint, attribs, expect);
         if (expect != VA_STATUS_SUCCESS) continue;
 
-        context = createContext(config, 1, 1);
+        context = createContext(config, min_resolution_width_height, min_resolution_width_height);
```

Comment:
Please wrap this line and all similar ones below.

Reply:
@uartie ,i am not sure what is the idea of wrap? would you explain it?

Reply:
thanks